RedHat-UX/red-hat-design-system
# docs(tag): update docs

#1019 

## What I did

### Tag
1.  Added the status table to overview tab
2. Updated images code to remove width
3. Made edits located in #1019 


## Testing Instructions

1. Please take a look at the deploy preview docs for `rh-tag` and verify that the edits look correct from the issue.
